docs/guides/platform/object-storage/using-object-storage-with-akamai-cdn/index.md

```diff
@@ -0,0 +1,101 @@
+
+-   **Scalability:** To preserve infrastructure integrity, object storage systems may have limitations in place for concurrent connections and simultaneous user streams. This can impact the ability to efficiently handle spikes in demand.
+
+-   **Retrieval speed:** Without CDN edge-based caching, content streamed directly from object storage origin servers may result in slower data retrieval speeds and higher latency.
+
+-   **Data transfer and egress costs:** Streaming directly from object storage can incur high egress costs since each user request may result in data transfer fees, whereas CDNs cache data at the edge to optimize delivery.
+
+-   **Resource limits:** Object storage systems may have limitations on the size of individual objects, potentially restricting the size of videos that can be efficiently streamed.
+
+-   **Rate limits:** Object storage solutions often have rate limits imposed on a requests per second (RPS) basis. If users are accessing Object Storage endpoints directly without the use of a CDN, there is a greater chance RPS limits are reached as more users access content.
+
+-   **Updating manifests:** Bucket contents can be determined by syncing with a manifest. This means that when source manifests are updated often, buckets may constantly have their contents changed.
+
+### Internet Considerations
+
+There are multiple internet-related factors to consider when building out a content delivery solution:
+
+-   **First mile:** The content provider connection point. Should content providers set up content to be accessible from a single physical location (i.e. an origin data center), user access is limited by “first mile” connectivity limitations.
+
+-   **Last mile:** The end-user connection point. Slow user internet connection speeds are uncontrollable and unpredictable. They can also mask or obfuscate problems related to the “first mile”, network peering, and bottleneck issues.
+
+-   **Peering points:** The connection points between two networks. These connections are not guaranteed and are difficult to troubleshoot. Networks benefit financially from connectivity with content providers and end users but not each other.
+
+-   **Backbone:** The infrastructure and physical connections that make up the internet. Hardware and network capacity limitations can cause performance issues.
+
+### Data Center-Based Architectures
+
+Consider the data center-based architecture below. Without the use of a CDN, object storage content distribution is limited to individual, origin data centers. Data center-based delivery relies heavily on public networks to pull and deliver content from regional origin points. At scale, this may result in bandwidth issues, high latency, and inefficient experiences for large numbers of users.
+
+![DC-Based Architecture](DC-Based-Architecture.png)
+
+## Advantages of Using Akamai CDN
```

Comment:
This section, except for the last sentence, is about advantages of using a CDN generally--not Akamai's CDN specifically

Reply:
changed to "Advantages of Using a CDN"